src/app/main/component/shared/components/input-google-autocomplete/input-google-autocomplete.component.ts

```diff
@@ -110,8 +110,11 @@ export class InputGoogleAutocompleteComponent implements OnInit, OnDestroy, Cont
           ...this.autoCompRequest
         };
 
-        this.autocompleteService.getPlacePredictions(request, (cityPredictionList) => {
-          this.predictionList = cityPredictionList?.filter((city) => !regex.test(city.description)) ?? [];
+        this.autocompleteService.getPlacePredictions(request, (predictions: google.maps.places.AutocompletePrediction[]) => {
+          predictions =
+            predictions?.filter((prediction) => !regex.test(prediction.description) && !prediction.terms[0].value.includes('вул.')) ?? [];
```

Comment:
Why terms[0] and not a description?

Reply:
All words from search are placed in `terms` array and "street" word is placed on zeros index. So why look up to the whole description string it appears only in the first element of `terms` array?